lib/govspeak.rb

```diff
@@ -279,18 +283,6 @@ def render_image(image)
     # More specific tags must be defined first. Those defined earlier have a
     # higher precedence for being matched. For example $CTA must be defined
     # before $C otherwise the first ($C)TA fill be matched to a contact tag.
-    wrap_with_div("call-to-action", "$CTA", Govspeak::Document)
-    wrap_with_div("summary", "$!")
-    wrap_with_div("form-download", "$D")
-    wrap_with_div("contact", "$C")
-    wrap_with_div("place", "$P", Govspeak::Document)
-    wrap_with_div("information", "$I", Govspeak::Document)
-    wrap_with_div("additional-information", "$AI")
-    wrap_with_div("example", "$E", Govspeak::Document)
-
-    extension("address", surrounded_by("$A")) do |body|
-      %(\n<div class="address"><div class="adr org fn"><p>\n#{body.sub("\n", '').gsub("\n", '<br />')}\n</p></div></div>\n)
-    end
```

Comment:
I would move the comment too, either to follow the wraps, or to go to the top of extensions - your call.

Reply:
I thought it would be weird if the comment saying some things should go first wasn't with the first things - i don't feel that strongly about it though